api/workers/index.js

```diff
@@ -97,23 +101,24 @@ function pagesWorker(connection) {
 
   const archiveBuildLogsQueue = new ArchiveBuildLogsQueue(connection);
   const buildTasksQueue = new BuildTasksQueue(connection);
+  const deleteOlderBuildsQueue = new DeleteOlderBuildsQueue(connection);
   const failStuckBuildsQueue = new FailStuckBuildsQueue(connection);
   const nightlyBuildsQueue = new NightlyBuildsQueue(connection);
   const scheduledQueue = new ScheduledQueue(connection);
   const timeoutBuildTasksQueue = new TimeoutBuildTasksQueue(connection);
   const queues = [
     archiveBuildLogsQueue,
     buildTasksQueue,
+    deleteOlderBuildsQueue,
     failStuckBuildsQueue,
     nightlyBuildsQueue,
     scheduledQueue,
     timeoutBuildTasksQueue,
   ];
 
   const jobs = () => Promise.all([
-    appEnv === 'production'
-      ? archiveBuildLogsQueue.add('archiveBuildLogsDaily', {}, nightlyJobConfig)
-      : Promise.resolve(),
+    archiveBuildLogsQueue.add('archiveBuildLogsDaily', {}, nightlyJobConfig),
```

Comment:
The only change I have made is this provisional code change. I didn't realize the buckets didn't exist, so that probably has to happen.

But FWIW the bucket service is already parameterized in `manifest.yml` as `federalist-((env))-s3-build-logs`. Would that not allow for the presence of these dev and staging buckets?

Reply:
it looks like non of the archive build log jobs were successful in dev so I was flagging as a possible culprit.

Reply:
Just checked dev and build logs aren't archiving nightly and builds over 180 days aren't deleting.

Reply:
True. It also looks like the migration on this PR hasn't run:
```
[...] column Build.deletedAt does not exist 
```